### ISSUE TITLE
verbose output

### ISSUE THREAD

**Issue by glpzzz** [daxslab/cuba-weather#16]
on param -v present output with the explanatory text and without it, just the value, one per line: Example

```
> ./weather` -vt Cienfuegos
Temperature: 17.41°C
> ./weather` -t Cienfuegos
17.41
```

**Comment by leynier** [comment]
Is this Issue a request for a new feature?

**Comment by glpzzz** [comment]
Yes, but we need to think it better. -v is already used for version. We need another letter and concept. 

I'm thinking more on a formatting like thing. For example -f "temp: /t grados" will output "temp: 35 grados". Un this way we can create simple and complex outputs. 

Ideas?
